ngraph/test/models/onnx/quantize_linear_zero_point.prototxt

```diff
@@ -8,6 +8,11 @@ graph {
     output: "y"
     name: "node1"
     op_type: "QuantizeLinear"
+    attribute {
+      name: "axis"
+      i: 0
+      type: INT
+    }
```

Comment:
How is it relevant to the PR subject?

Reply:
Generally my change exposed an undefined behavior in onnx importer and this was one of the failing tests on Ubuntu 32-bit Release mode. I tried to solve this issue from the wrong angle while actually this https://github.com/openvinotoolkit/openvino/pull/2064/files#diff-70e86a8186be5fd971e74b665f52ff53R40 was the correct fix. I will revert that change to the model.